package.json

```
@@ -4,12 +4,22 @@
   "private": true,
   "description": "Учебный проект «Угадай мелодию» от HTML Academy",
   "scripts": {
-    "eslint": "eslint src/**/*.js",
+    "build":"webpack --mode production",
+    "start":"webpack-dev-server",
```

nit, старайся соблюдать одинаковый стиль во всех джейсонах.
Здесь и выше просится пробел после названия таски.

```suggestion
    "start": "webpack-dev-server",
```